composer.json

```diff
@@ -29,14 +30,14 @@
         "twig/twig": "~1.12|~2.0",
         "doctrine/orm": "~2.1",
         "doctrine/phpcr-odm": "^1.3",
-        "jackalope/jackalope-doctrine-dbal": "^1.0",
+        "jackalope/jackalope-doctrine-dbal": "^1.1.5",
         "propel/propel1": "~1.7",
-        "symfony/yaml": "2.*",
-        "symfony/translation": "~2.0",
+        "symfony/yaml": "^2.0.5",
```

Comment:
IIRC, 2.0.6 also had broken metadata, so the lowest dependency should be higher (I suggest making it at least 2.1, like other components)